INFO     TestDemoBlazeE2E_edge:test_demoblaze_e2e.py:1142 ==================================================
INFO     TestDemoBlazeE2E_edge:test_demoblaze_e2e.py:1143 Starting new test on EDGE
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:63 Selenium Manager binary found at: /opt/hostedtoolcache/Python/3.9.23/x64/lib/python3.9/site-packages/selenium/webdriver/common/linux/selenium-manager
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:120 Executing process: /opt/hostedtoolcache/Python/3.9.23/x64/lib/python3.9/site-packages/selenium/webdriver/common/linux/selenium-manager --browser MicrosoftEdge --debug --output json
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:137 Found msedgedriver 140.0.3485.94 in PATH: /usr/bin/msedgedriver
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:137 MicrosoftEdge detected at /usr/bin/microsoft-edge
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:137 Running command: /usr/bin/microsoft-edge --version
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:137 Output: "Microsoft Edge 141.0.3537.99 "
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:137 Detected browser: MicrosoftEdge 141.0.3537.99
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:137 Reading msedgedriver version from https://msedgedriver.azureedge.net/LATEST_RELEASE_141_LINUX
WARNING  selenium.webdriver.common.selenium_manager:selenium_manager.py:135 Exception managing MicrosoftEdge: error sending request for url (https://msedgedriver.azureedge.net/LATEST_RELEASE_141_LINUX): error trying to connect: dns error: failed to lookup address information: Name does not resolve
WARNING  selenium.webdriver.common.selenium_manager:selenium_manager.py:135 The msedgedriver version (140.0.3485.94) detected in PATH at /usr/bin/msedgedriver might not be compatible with the detected MicrosoftEdge version (141.0.3537.99); currently, msedgedriver  is recommended for MicrosoftEdge 141.*, so it is advised to delete the driver in PATH and retry
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:137 Driver path: /usr/bin/msedgedriver
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:137 Browser path: /usr/bin/microsoft-edge
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:98 Using driver at: /usr/bin/msedgedriver
DEBUG    selenium.webdriver.common.service:service.py:215 Started executable: `/usr/bin/msedgedriver` in a child process with pid: 4644
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:313 POST http://localhost:32959/session {"capabilities": {"firstMatch": [{}], "alwaysMatch": {"browserName": "MicrosoftEdge", "pageLoadStrategy": "normal", "browserVersion": null, "ms:edgeOptions": {"prefs": {"download.default_directory": "/home/runner/work/selenium-demoblaze-operations/selenium-demoblaze-operations/downloads", "download.prompt_for_download": false, "download.directory_upgrade": true, "safebrowsing.enabled": false, "profile.default_content_settings.popups": 0, "profile.default_content_setting_values.notifications": 2}, "excludeSwitches": ["enable-logging", "enable-automation"], "useAutomationExtension": false, "extensions": [], "binary": "/usr/bin/microsoft-edge", "args": ["--headless=new", "--window-size=1920,1080", "--start-maximized", "--disable-gpu", "--no-sandbox", "--disable-dev-shm-usage", "--disable-software-rasterizer", "--disable-blink-features=AutomationControlled", "--single-process", "--disable-features=VizDisplayCompositor", "--disable-setuid-sandbox", "--disable-extensions", "--disable-popup-blocking", "--ignore-certificate-errors"]}}}}
DEBUG    urllib3.connectionpool:connectionpool.py:244 Starting new HTTP connection (1): localhost:32959
DEBUG    urllib3.connectionpool:connectionpool.py:546 http://localhost:32959 "POST /session HTTP/1.1" 200 0
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:329 Remote response: status=200 | data={"value":{"capabilities":{"acceptInsecureCerts":false,"browserName":"MicrosoftEdge","browserVersion":"141.0.3537.99","fedcm:accounts":true,"ms:edgeOptions":{"debuggerAddress":"localhost:38455"},"msedge":{"msedgedriverVersion":"140.0.3485.94 (9831641b47fbddfcc786ceb18c2e499530dc5e2b)","userDataDir":"/tmp/.com.microsoft.Edge.63N4Tx"},"networkConnectionEnabled":false,"pageLoadStrategy":"normal","platformName":"linux","proxy":{},"setWindowRect":true,"strictFileInteractability":false,"timeouts":{"implicit":0,"pageLoad":300000,"script":30000},"unhandledPromptBehavior":"dismiss and notify","webauthn:extension:credBlob":true,"webauthn:extension:largeBlob":true,"webauthn:extension:minPinLength":true,"webauthn:extension:prf":true,"webauthn:virtualAuthenticators":true},"sessionId":"44ef8f592afe0b6955b3321d158472a2"}} | headers=HTTPHeaderDict({'Content-Length': '817', 'Content-Type': 'application/json; charset=utf-8', 'cache-control': 'no-cache'})
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:356 Finished Request
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:313 POST http://localhost:32959/session/44ef8f592afe0b6955b3321d158472a2/timeouts {"implicit": 10000}
DEBUG    urllib3.connectionpool:connectionpool.py:546 http://localhost:32959 "POST /session/44ef8f592afe0b6955b3321d158472a2/timeouts HTTP/1.1" 200 0
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:329 Remote response: status=200 | data={"value":null} | headers=HTTPHeaderDict({'Content-Length': '14', 'Content-Type': 'application/json; charset=utf-8', 'cache-control': 'no-cache'})
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:356 Finished Request
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:313 POST http://localhost:32959/session/44ef8f592afe0b6955b3321d158472a2/timeouts {"pageLoad": 60000}
DEBUG    urllib3.connectionpool:connectionpool.py:546 http://localhost:32959 "POST /session/44ef8f592afe0b6955b3321d158472a2/timeouts HTTP/1.1" 200 0
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:329 Remote response: status=200 | data={"value":null} | headers=HTTPHeaderDict({'Content-Length': '14', 'Content-Type': 'application/json; charset=utf-8', 'cache-control': 'no-cache'})
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:356 Finished Request
DEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.address`.
DEBUG    faker.factory:factory.py:95 Provider `faker.providers.address` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.automotive`.
DEBUG    faker.factory:factory.py:95 Provider `faker.providers.automotive` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.bank`.
DEBUG    faker.factory:factory.py:86 Specified locale `en_US` is not available for provider `faker.providers.bank`. Locale reset to `en_GB` for this provider.
DEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.barcode`.
DEBUG    faker.factory:factory.py:95 Provider `faker.providers.barcode` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.color`.
DEBUG    faker.factory:factory.py:95 Provider `faker.providers.color` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.company`.
DEBUG    faker.factory:factory.py:95 Provider `faker.providers.company` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.credit_card`.
DEBUG    faker.factory:factory.py:95 Provider `faker.providers.credit_card` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.currency`.
DEBUG    faker.factory:factory.py:95 Provider `faker.providers.currency` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.date_time`.
DEBUG    faker.factory:factory.py:95 Provider `faker.providers.date_time` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:106 Provider `faker.providers.emoji` does not feature localization. Specified locale `en_US` is not utilized for this provider.
DEBUG    faker.factory:factory.py:106 Provider `faker.providers.file` does not feature localization. Specified locale `en_US` is not utilized for this provider.
DEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.geo`.
DEBUG    faker.factory:factory.py:95 Provider `faker.providers.geo` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.internet`.
DEBUG    faker.factory:factory.py:95 Provider `faker.providers.internet` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:106 Provider `faker.providers.isbn` does not feature localization. Specified locale `en_US` is not utilized for this provider.
DEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.job`.
DEBUG    faker.factory:factory.py:95 Provider `faker.providers.job` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.lorem`.
DEBUG    faker.factory:factory.py:95 Provider `faker.providers.lorem` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.misc`.
DEBUG    faker.factory:factory.py:95 Provider `faker.providers.misc` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.passport`.
DEBUG    faker.factory:factory.py:95 Provider `faker.providers.passport` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.person`.
DEBUG    faker.factory:factory.py:95 Provider `faker.providers.person` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.phone_number`.
DEBUG    faker.factory:factory.py:95 Provider `faker.providers.phone_number` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:106 Provider `faker.providers.profile` does not feature localization. Specified locale `en_US` is not utilized for this provider.
DEBUG    faker.factory:factory.py:106 Provider `faker.providers.python` does not feature localization. Specified locale `en_US` is not utilized for this provider.
DEBUG    faker.factory:factory.py:106 Provider `faker.providers.sbn` does not feature localization. Specified locale `en_US` is not utilized for this provider.
DEBUG    faker.factory:factory.py:76 Looking for locale `en_US` in provider `faker.providers.ssn`.
DEBUG    faker.factory:factory.py:95 Provider `faker.providers.ssn` has been localized to `en_US`.
DEBUG    faker.factory:factory.py:106 Provider `faker.providers.user_agent` does not feature localization. Specified locale `en_US` is not utilized for this provider.
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:313 POST http://localhost:32959/session/44ef8f592afe0b6955b3321d158472a2/url {"url": "https://www.demoblaze.com"}
DEBUG    urllib3.connectionpool:connectionpool.py:546 http://localhost:32959 "POST /session/44ef8f592afe0b6955b3321d158472a2/url HTTP/1.1" 200 0
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:329 Remote response: status=200 | data={"value":null} | headers=HTTPHeaderDict({'Content-Length': '14', 'Content-Type': 'application/json; charset=utf-8', 'cache-control': 'no-cache'})
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:356 Finished Request
INFO     TestDemoBlazeE2E_edge:test_demoblaze_e2e.py:1163 Navigated to DemoBlaze website on edge
INFO     TestDemoBlazeE2E_edge:test_demoblaze_e2e.py:1395 Testing cookie operations
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:313 GET http://localhost:32959/session/44ef8f592afe0b6955b3321d158472a2/cookie {}
DEBUG    urllib3.connectionpool:connectionpool.py:546 http://localhost:32959 "GET /session/44ef8f592afe0b6955b3321d158472a2/cookie HTTP/1.1" 200 0
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:329 Remote response: status=200 | data={"value":[]} | headers=HTTPHeaderDict({'Content-Length': '12', 'Content-Type': 'application/json; charset=utf-8', 'cache-control': 'no-cache'})
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:356 Finished Request
INFO     TestDemoBlazeE2E_edge:base_page.py:293 Got 0 cookies
INFO     TestDemoBlazeE2E_edge:test_demoblaze_e2e.py:1398 Found 0 cookies
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:313 POST http://localhost:32959/session/44ef8f592afe0b6955b3321d158472a2/cookie {"cookie": {"name": "test_cookie", "value": "test_value_123", "path": "/", "domain": ".demoblaze.com"}}
DEBUG    urllib3.connectionpool:connectionpool.py:546 http://localhost:32959 "POST /session/44ef8f592afe0b6955b3321d158472a2/cookie HTTP/1.1" 200 0
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:329 Remote response: status=200 | data={"value":null} | headers=HTTPHeaderDict({'Content-Length': '14', 'Content-Type': 'application/json; charset=utf-8', 'cache-control': 'no-cache'})
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:356 Finished Request
INFO     TestDemoBlazeE2E_edge:base_page.py:299 Added cookie: {'name': 'test_cookie', 'value': 'test_value_123', 'path': '/', 'domain': '.demoblaze.com'}
INFO     TestDemoBlazeE2E_edge:test_demoblaze_e2e.py:1407 Added custom cookie
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:313 POST http://localhost:32959/session/44ef8f592afe0b6955b3321d158472a2/refresh {}
DEBUG    urllib3.connectionpool:connectionpool.py:546 http://localhost:32959 "POST /session/44ef8f592afe0b6955b3321d158472a2/refresh HTTP/1.1" 200 0
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:329 Remote response: status=200 | data={"value":null} | headers=HTTPHeaderDict({'Content-Length': '14', 'Content-Type': 'application/json; charset=utf-8', 'cache-control': 'no-cache'})
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:356 Finished Request
INFO     TestDemoBlazeE2E_edge:base_page.py:314 Page refreshed
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:313 GET http://localhost:32959/session/44ef8f592afe0b6955b3321d158472a2/cookie {}
DEBUG    urllib3.connectionpool:connectionpool.py:546 http://localhost:32959 "GET /session/44ef8f592afe0b6955b3321d158472a2/cookie HTTP/1.1" 200 0
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:329 Remote response: status=200 | data={"value":[{"domain":".demoblaze.com","httpOnly":false,"name":"test_cookie","path":"/","sameSite":"Lax","secure":true,"value":"test_value_123"},{"domain":"www.demoblaze.com","httpOnly":false,"name":"user","path":"/","sameSite":"Lax","secure":false,"value":"7200b6a3-9612-fb0e-33fa-96f14448c51b"}]} | headers=HTTPHeaderDict({'Content-Length': '296', 'Content-Type': 'application/json; charset=utf-8', 'cache-control': 'no-cache'})
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:356 Finished Request
INFO     TestDemoBlazeE2E_edge:base_page.py:293 Got 2 cookies
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:313 DELETE http://localhost:32959/session/44ef8f592afe0b6955b3321d158472a2/cookie/test_cookie {}
DEBUG    urllib3.connectionpool:connectionpool.py:546 http://localhost:32959 "DELETE /session/44ef8f592afe0b6955b3321d158472a2/cookie/test_cookie HTTP/1.1" 200 0
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:329 Remote response: status=200 | data={"value":null} | headers=HTTPHeaderDict({'Content-Length': '14', 'Content-Type': 'application/json; charset=utf-8', 'cache-control': 'no-cache'})
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:356 Finished Request
INFO     TestDemoBlazeE2E_edge:base_page.py:304 Deleted cookie: test_cookie
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:313 DELETE http://localhost:32959/session/44ef8f592afe0b6955b3321d158472a2/cookie {}
DEBUG    urllib3.connectionpool:connectionpool.py:546 http://localhost:32959 "DELETE /session/44ef8f592afe0b6955b3321d158472a2/cookie HTTP/1.1" 200 0
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:329 Remote response: status=200 | data={"value":null} | headers=HTTPHeaderDict({'Content-Length': '14', 'Content-Type': 'application/json; charset=utf-8', 'cache-control': 'no-cache'})
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:356 Finished Request
INFO     TestDemoBlazeE2E_edge:base_page.py:309 Deleted all cookies
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:313 GET http://localhost:32959/session/44ef8f592afe0b6955b3321d158472a2/cookie {}
DEBUG    urllib3.connectionpool:connectionpool.py:546 http://localhost:32959 "GET /session/44ef8f592afe0b6955b3321d158472a2/cookie HTTP/1.1" 200 0
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:329 Remote response: status=200 | data={"value":[]} | headers=HTTPHeaderDict({'Content-Length': '12', 'Content-Type': 'application/json; charset=utf-8', 'cache-control': 'no-cache'})
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:356 Finished Request
INFO     TestDemoBlazeE2E_edge:base_page.py:293 Got 0 cookies
INFO     TestDemoBlazeE2E_edge:test_demoblaze_e2e.py:1417 Cookies after deletion: 0
INFO     TestDemoBlazeE2E_edge:test_demoblaze_e2e.py:1168 Test completed on edge
INFO     TestDemoBlazeE2E_edge:test_demoblaze_e2e.py:1169 ==================================================
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:313 DELETE http://localhost:32959/session/44ef8f592afe0b6955b3321d158472a2 {}
DEBUG    urllib3.connectionpool:connectionpool.py:546 http://localhost:32959 "DELETE /session/44ef8f592afe0b6955b3321d158472a2 HTTP/1.1" 200 0
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:329 Remote response: status=200 | data={"value":null} | headers=HTTPHeaderDict({'Content-Length': '14', 'Content-Type': 'application/json; charset=utf-8', 'cache-control': 'no-cache'})
DEBUG    selenium.webdriver.remote.remote_connection:remote_connection.py:356 Finished Request